docs/computing/confidential/requirements.md

```diff
@@ -25,22 +25,32 @@ following [the documentation here](../../tools/aleph-client/index.md).
 ### sevctl
 
 Installing [Rust and Cargo](https://doc.rust-lang.org/cargo/getting-started/installation.html):
+
 ```shell
 curl https://sh.rustup.rs -sSf | sh
 ```
 
-or on Ubuntu 24.04:
+Or on Ubuntu 24.04:
 
-```
+```shell
 apt install cargo
 ```
 
+On some distributions (ex: Ubuntu), sevctl requires additional dependencies:
+
+```shell
+apt install -y pkg-config libssl-dev asciidoctor
+```
+
```

Comment:
I think that with `rustup` installation, is not needed to install cargo again from dependencies.